common/src/Jamfile

```diff
@@ -83,7 +83,7 @@ LIBPATHS =  ;
 #	additional paths to look for system headers
 #	thes use the form: #include <header>
 #	source file directories are NOT auto-included here
-SYSTEM_INCLUDE_PATHS = /system/develop/headers/x86/liblayout/
```

Comment:
Won't this break compiling on x86_ggc2?

Reply:
No, this was a mistake from when I was testing the GCC7 build on 32-bit. The header directories are theoretically identical.

Reply:
I misread the change, I thought "/x86/" was added, not removed. Makes sense now.